src/interfaces/registry.ts

```diff
@@ -4,29 +4,44 @@
 
 import { RegistryListing } from 'computable/dist/interfaces'
 import { Nos } from 'computable/dist/types'
+import { Map } from '../interfaces'
+
+interface Registry {
+  // deployed address of the registry contract, can be falsy at first
+  address?: string
+  applicants?: Applicant[]
+  challenges?: Challenge[]
+  listings?: Listing[]
+}
 
 // our shape for applicant is simply the 3 args required for the apply method
-export interface Applicant {
-  name: string; // this can be just about anynthing. NOTE: will need to be hashed on calling registry.apply by using the helper `stringToBytes`
-  deposit?: Nos; // will default to the paramaterizer MIN_DEPOSIT if excluded
-  data?: string; // again, just about anything can go here.
+interface Applicant {
+  name: string // this can be just about anynthing. NOTE: will need to be hashed on calling registry.apply by using the helper `stringToBytes`
+  deposit?: Nos // will default to the paramaterizer MIN_DEPOSIT if excluded
+  data?: string // again, just about anything can go here.
 }
 
-export interface Challenge {
+interface ApplicantData {
```

Comment:
does TSC not throw errors when building with non-exported interfaces?

It used too